src/hexo/themes/sonarwhal/layout/partials/footer.hbs

```diff
@@ -25,6 +25,11 @@
                     <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 24"><path class="twitter-icon-svg" d="M25 4.4c-.9.4-1.9.7-2.9.8 1-.6 1.8-1.6 2.2-2.8-1 .6-2.1 1-3.2 1.2-.9-1-2.2-1.6-3.7-1.6-2.8 0-5 2.3-5 5 0 .4 0 .8.1 1.2C8.3 8 4.6 6 2.1 2.9c-.4.8-.7 1.6-.7 2.6 0 1.8.9 3.3 2.2 4.2-.8-.1-1.5-.3-2.2-.7v.1c0 2.4 1.7 4.5 4.1 5-.4.1-.9.2-1.3.2-.3 0-.6 0-.9-.1.6 2 2.5 3.5 4.7 3.5-1.9 1.3-4 2.1-6.4 2.1-.4 0-.8 0-1.2-.1C2.6 21.2 5.3 22 8.1 22c9.3 0 14.4-7.7 14.4-14.4v-.7c1-.6 1.8-1.5 2.5-2.5"/></svg>
                 </a>
             </li>
+            <li class="medium-icon">
+                <a href="https://medium.com/sonarwhal" aria-label="Follow Sonar on Medium">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 45 45"><path class="medium-icon-svg" d="M5 40V5h35v35H5zm8.56-12.627c0 .555-.027.687-.318 1.03l-2.457 2.985v.396h6.974v-.396l-2.456-2.985c-.291-.343-.344-.502-.344-1.03V18.42l6.127 13.364h.714l5.256-13.364v10.644c0 .29 0 .342-.185.528l-1.848 1.796v.396h9.19v-.396l-1.822-1.796c-.184-.186-.21-.238-.21-.528V15.937c0-.291.026-.344.21-.528l1.823-1.797v-.396h-6.471l-4.622 11.542-5.203-11.542h-6.79v.396l2.14 2.64c.239.292.291.37.291.768v10.353z"/></svg>
```

Comment:
This can be even smaller:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 45 45"><path d="M5 40V5h35v35H5zm8.56-12.63c0 .56-.03.7-.32 1.03l-2.45 3v.38h6.9v-.4l-2.4-2.98c-.3-.34-.35-.5-.35-1.03v-8.95l6.13 13.36h.7L27 18.42v10.64c0 .3 0 .35-.2.53L25 31.4v.37h9.2v-.4l-1.82-1.8c-.2-.17-.22-.23-.22-.52v-13.1c0-.3.03-.36.2-.54l1.83-1.8v-.32h-6.5L23.1 24.8l-5.2-11.55h-6.8v.4l2.14 2.63c.24.3.3.37.3.77V27.4z" class="medium-icon-svg"/></svg>
```

Reply:
How did you do that?

Reply:
I used https://jakearchibald.github.io/svgomg/.

Reply:
Awesome tool, thanks!